zzzz_modular_occulus/code/game/objects/items/weapons/storage/pouches.dm

```diff
@@ -44,3 +44,17 @@
 
 /obj/item/weapon/tool/sword
 	w_class = ITEM_SIZE_BULKY
+
+/obj/item/weapon/storage/pouch/fanny
+	name = "fannypack"
+	desc = "An economical solution for general purpose belt-located storage. Holds less than a large pouch."
```

Comment:
If it holds less than a large pouch on your waist, it's worthless. Just saying.

Reply:
Irrelevant.